### PR TITLE
Re-enable app check platform logging

### PR DESCRIPTION
**Pull request by ncooke3**
b/185793743

### CODE REVIEW

File: FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.m [hunk 1]
```diff
@@ -111,18 +111,11 @@ - (instancetype)initWithURLSession:(NSURLSession *)session
 
              [request setValue:self.APIKey forHTTPHeaderField:kAPIKeyHeaderKey];
 
-             NSString *versionHeader =
-                 [NSString stringWithFormat:@"fire-app-check/%@", FIRFirebaseVersion()];
-             [request setValue:versionHeader forHTTPHeaderField:kUserAgentKey];
```
Comment (ncooke3) [inline FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.m:116]: Double checking, this version header is intended to be removed, correct?

Comment (maksymmalyhin) [reply FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.m:116]: Yes, it's correct.

**Review by maksymmalyhin — APPROVED** [review]
👍  Thank you for the change!

**Review by paulb777 — APPROVED** [review]

**Comment by ncooke3** [comment]
Performed a manual end-to-end test to confirm SDK properly interacts with server.

**Pull request merged**